config/bench.exs

```diff
@@ -4,3 +4,4 @@ config :t, T.Media.Client, adapter: T.Media.S3Client
 config :t, T.Bot, adapter: T.Bot.API
 config :t, T.PushNotifications.APNS, adapter: T.PushNotifications.APNS.FinchAdapter
 config :t, T.Twilio, adapter: T.Twilio.HTTP
+config :t, T.Accounts.AppleSignIn, adapter: T.Accounts.AppleSignIn.HTTPAdapter
```

Comment:
this fixes benches compilation, it was failing without apple sign in adapter set since it's required during compilation (but it's not actually used in benches)